bin/si-gen-cloud-control/src/pipeline-steps/generateAssetFuncs.ts

```diff
@@ -19,7 +19,7 @@ export function generateAssetFuncs(specs: PkgSpec[]): PkgSpec[] {
       continue;
     }
 
-    const assetFuncUniqueKey = schemaVariant.data.funcUniqueId;
+    const assetFuncUniqueKey = schemaVariant.data?.funcUniqueId;
```

Comment:
Data shouldn't be optional on this, if it really is, we should change the type definition. (but I believe you can just drop the `?`)

Reply:
The tests will complain without this check

```
error: TS18047 [ERROR]: 'schemaVariant.data' is possibly 'null'.
    const assetFuncUniqueKey = schemaVariant.data.funcUniqueId;
                               ~~~~~~~~~~~~~~~~~~
    at file:///home/scott/projects/si/bin/si-gen-cloud-control/src/pipeline-steps/generateAssetFuncs.ts:24:32
```

Reply:
Yeah, looking at the spec, data is nullable, so we need to check that it exists when we check the schemaVariant

Reply:
Ok, my IDE is messing something up then, sorry